README.md

```diff
@@ -13,80 +13,83 @@ Debug | Release
+*Latest stable version:* [1.1.0](https://www.nuget.org/packages/Desktop.Analyzers/)
 
+Analyzers for APIs specific to the desktop .NET Framework.
 
-Microsoft.AnalyzerPowerPack
----------------------------
+[More info](src/Desktop.Analyzers/Desktop.Analyzers.md)
 
-*Latest stable version:* [1.0.1](https://www.nuget.org/packages/Microsoft.AnalyzerPowerPack/)
+Microsoft.ApiDesignGuidelines.Analyzers
+--------------------------------
 
-General language rules implemented as analyzers using the .NET Compiler Platform ("Roslyn").
+Provides analyzers which implement the .NET design guidelines for API authors.
 
-[More info](src/AnalyzerPowerPack/AnalyzerPowerPack.md)
+[More info](src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md)
 
 
 Microsoft.CodeAnalysis.Analyzers
 --------------------------------
 
-*Latest stable version:* [1.0.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/)
+*Latest stable version:* [1.1.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/)
 
 Provides guidelines for using .NET Compiler Platform ("Roslyn") APIs.
 
 [More info](src/CodeAnalysis/Microsoft.CodeAnalysis.Analyzers.md)
 
 
-Microsoft.Net.RoslynDiagnostics
+Roslyn.Diagnostics.Analyzers
 -------------------------------
 
 Contains analyzers specific to the .NET Compiler Platform ("Roslyn") project.
 
-[More info](src/Roslyn/Microsoft.Net.RoslynDiagnostics.md)
+[More info](src/Roslyn.Diagnostics.Analyzers/Roslyn.Diagnostics.Analyzers.md)
+
+System.Collections.Immutable.Analyzers
+------------------------
 
+Provides guidelines for using APIs in the System.Collections.Immutable contract.
+
+[More info](src/System.Collections.Immutable.Analyzers/System.Collections.Immutable.Analyzers.md)
 
 System.Runtime.Analyzers
 ------------------------
 
-*Latest stable version:* [1.0.1](https://www.nuget.org/packages/System.Runtime.Analyzers/)
+*Latest stable version:* [1.1.0](https://www.nuget.org/packages/System.Runtime.Analyzers/)
 
-Provides guidelines for using APIs in the System.Runtime namespace.
+Provides guidelines for using APIs in the System.Runtime contract.
 
-[More info](src/FxCop/System.Runtime.Analyzers/System.Runtime.Analyzers.md)
+[More info](src/System.Runtime.Analyzers/System.Runtime.Analyzers.md)
 
 
 System.Runtime.InteropServices.Analyzers
 ----------------------------------------
 
-*Latest stable version:* [1.0.1](https://www.nuget.org/packages/System.Runtime.InteropServices.Analyzers/)
+*Latest stable version:* [1.1.0](https://www.nuget.org/packages/System.Runtime.InteropServices.Analyzers/)
+
+Provides guidelines for using APIs in the System.Runtime.InteropServices contract.
+
+[More info](src/System.Runtime.InteropServices.Analyzers/System.Runtime.InteropServices.Analyzers.md)
+
+System.Security.Cryptography.Hashing.Algorithms.Analyzers
+------------------------
```

Comment:
The dashes have the effect of rendering the previous line as a title. It just needs some dashes - so not extending to the end of the line doesn't matter.